src/components/PrintModal/PrintModal.js

```diff
@@ -464,14 +467,15 @@ class PrintModal extends React.PureComponent {
       return null;
     }
 
-    const { count, pagesToPrint } = this.state;
+    const { count, pagesToPrint, isProcessingPagesToPrint } = this.state;
     const className = getClassName('Modal PrintModal', this.props);
     const customPagesLabelElement = (
       <input
         ref={this.customInput}
         type="text"
         placeholder={t('message.customPrintPlaceholder')}
         onFocus={this.onFocus}
+        disabled={this.state.isProcessingPagesToPrint}
       />
     );
     const isPrinting = count >= 0;
```

Comment:
Is this variable the same as `this.state.isProcessingPagesToPrint`? If it is then we can just use this variable instead of adding a new state.

For example:
```javascript
                <Input
                  ref={this.currentPage}
                  id="current-page"
                  name="pages"
                  type="radio"
                  label={t('option.print.current')}
                  // disabled={isProcessingPagesToPrint}
                  disabled={isPrinting}
                />
```

Reply:
done. i'm guessing its because we don'ot want to force a whole rerender when i change the state?

Reply:
Not really. I find that deriving a state(`isPrinting`) from an existing state(`this.state.count`) is usually better than syncing two states(`this.state.count` and `this.state.isProcessingPagesToPrint`).

What I'm suggesting is that we can just get rid of `isProcessingPagesToPrint` and use the derived `isPrinting` in the render function, as the above code snippet showed. It seems to me that `isPrinting` is the same as `isProcessingPagesToPrint` so we can just use `isPrinting`.

Reply:
done. funny how i didn't see `isPrinting`.

I have a question if we go this route.
![image](https://user-images.githubusercontent.com/54084743/71831952-c21d9880-305e-11ea-814f-296425ec216c.png)
Should we set `this.state.count` to be -1 on error?

I don't know how to force an error. But I'm guessing it will remain disabled even on error which could be a problem in terms of user experience.

Reply:
Uhh I missed this comment. Yeah we should set it to -1 in the catch block. I will do it.